src/js/components/Menu.js

```diff
@@ -163,6 +163,7 @@ class MenuDrop extends Component {
     } = this.props;
     const restProps = Props.omit(props,
       Object.keys(MenuDrop.childContextTypes));
+    delete restProps.dropColorIndex;
```

Comment:
should we do this instead?

restProps =  Props.omit(restProps,
        Object.keys(MenuDrop.propTypes));